workshops/views.py

```diff
@@ -521,6 +521,19 @@ def person_password(request, person_id):
     })
 
 
+@login_required
+def person_merge(request):
+    '''Merge information from one Person into another (in case of duplicates).'''
+
+    person_merge_form = PersonMergeForm()
+
+    if request.method == 'POST':
+        pass # FIXME
+
+    context = {'title': 'Merge Persons',
+               'person_merge_form': person_merge_form}
```

Comment:
Add a form helper here so it [can be used in the template](https://github.com/swcarpentry/amy/pull/366/files#r32471693):

```python
    context = {'title': 'Merge Persons',
               'person_merge_form': person_merge_form,
               'form_helper': bootstrap_helper}
```

`bootstrap_helper` is an object accessible from all our views.